src/main/java/org/symphonyoss/symphony/messageml/elements/RoomSelector.java

```diff
@@ -84,9 +86,10 @@ public void validate() throws InvalidInputException {
     }
 
     if (getAttribute(VALUE_ATTR) != null) {
+      String attribute = getAttribute(VALUE_ATTR);
```

Comment:
not used ?

Reply:
the result is not used ,it's used just to validate the format of the list of string